src/dotnetTemplate/Microsoft.PowerShell.PowerShellStandard.Template/Microsoft.PowerShell.PowerShellStandard.Template/InvokeNoun.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Management.Automation;
+using System.Management.Automation.Runspaces;
+
+namespace Microsoft.PowerShell.PowerShellStandard.Template
+{
+    [Cmdlet("Invoke","Noun")]
```

Comment:
My preference would be to use the `Test` verb instead of `Invoke` as people may tend to leave it as `Invoke` but likely to change `Test`.  Also better to use the enum for the Verb than a string to enforce proper verb usage.

For the Noun, perhaps call it `SampleCmdlet` to make it more obvious they need to change it?